week-3/refactor/format-as-12-hours.js

```diff
@@ -4,3 +4,29 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+function formatAs12HourClock(time) {
+    const analog = Number(time.slice(0, 2))
+    if (analog > 12) {
+      return `${analog - 12}:00 pm`;
```

Comment:
with this solution, `21:38` will return `9:00 pm` which isnt right, it should be `09:38 pm`